torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -387,12 +387,21 @@ void ComputeAt::setupOutputs() {
     return;
 
   std::vector<TensorView*> touched_output_order;
+  const auto& terminating_outputs =
```

Comment:
do we need the "full" getTerminatingOutputs() here? I mean, it looks we just want the unordered set version of terminating outputs (which would also avoid the O(N^2) lookup)

Reply:
That's right. We could create a `unordered_set` version of the function. In practice, since the number of output tensors would be just a few, just a `vector` would be actually faster. In any case, since the number is pretty small, I don't think the efficiency would matter.